test/integration/authorization_test.go

```diff
@@ -1550,8 +1550,7 @@ func TestOldLocalResourceAccessReviewEndpoint(t *testing.T) {
 	}
 }
 
-// TestClusterPolicyCache confirms that the creation of cluster role bindings fallback to live lookups when the referenced cluster role is not cached
-func TestClusterPolicyCache(t *testing.T) {
```

Comment:
Why removed? Flaky?

Reply:
This was supposed to be removed as part of one of the earlier RBAC clean ups.  There is no policy, and no cache to test anymore.